source/slang/core.meta.slang

```diff
@@ -220,7 +222,7 @@ interface __BuiltinSignedArithmeticType : __BuiltinArithmeticType {}
 /// A type that can represent integers
 [sealed]
 [builtin]
-interface __BuiltinIntegerType : __BuiltinArithmeticType
+interface __BuiltinIntegerType : __BuiltinArithmeticType, IInteger
```

Comment:
I was wondering about this when I was drawing a hierarchy diagram.
https://github.com/shader-slang/slang/issues/4196#issuecomment-2121042634